gatsby-config.js

```diff
@@ -1,9 +1,12 @@
 module.exports = {
 	siteMetadata: {
 		title: `Etimo`,
-		description: `Vår vision: Sveriges ledande utvecklingsorganisation.`,
+		type: `website`,
 		author: `@etimo`,
-		siteUrl: 'https://etimo.se',
+		siteUrl: `https://etimo.se`,
+		description: `Bättre mjukvara för en bättre värld.`,
+		imageAlt: `Etimo logo`,
+		imageType: `image/png`
```

Comment:
Just a minor detail but use standard " around these strings instead of `. ` is for interpolated strings.

Reply:
Opted for ` instead since that was what was in use before I started. I'll updated it.